src/Aspire.Hosting.Azure/AzureEnvironmentResource.cs

```diff
@@ -82,15 +82,17 @@ public AzureEnvironmentResource(string name, ParameterResource location, Paramet
 
             var provisionStep = new PipelineStep
             {
-                Name = WellKnownPipelineSteps.ProvisionInfrastructure,
-                Action = ctx => ProvisionAzureBicepResourcesAsync(ctx, provisioningContext!)
+                Name = "provision-azure-bicep-resources",
```

Comment:
@captainsafia - do we still want `WellKnownPipelineSteps` as well?

Reply:
I was going to add this but then I realized that we'd be reworking this when we split up this step into multiple steps per resource so I wasn't sure that it was actually needed. We'll end up making the other "steps" more granular as well and applying tags to them. So, in reality, I think the utility of WellKnownPipelineSteps is not there anymore and tags resolve the issue of lookup.

Reply:
That makes sense